### PR TITLE
Feature/kx 80 login UI

### PR DESCRIPTION
**Pull request by GiuseppeArbore**
Consider if we want to keep the select or remove it

### CODE REVIEW

File: client/src/components/landing/HeroSection.tsx [hunk 1]
```diff
@@ -19,6 +23,29 @@ const HeroSection: React.FC = () => {
           </Badge>
         </Link>
       </div>
+      <Button onClick={() => setShowLogin(true)}>Login</Button>
+      <Dialog open={showLogin} onClose={(val) => setShowLogin(val)} static={true}>
+        <DialogPanel>
+          <h2 className="text-lg font-semibold text-tremor-content-strong dark:text-dark-tremor-content-strong">Login</h2>
+          <div className="mx-auto max-w-sm space-y-8 mt-2">
+            <div>
+              <TextInput placeholder="Type your username here" error={error} errorMessage="Wrong username" />
```
Comment (golnazsaraji) [inline client/src/components/landing/HeroSection.tsx:32]: If I leave the username empty, there is no error message, do you want to control the validations on another phase? if yes, that is ok!

File: client/src/components/landing/HeroSection.tsx [hunk 2]
```diff
@@ -19,6 +23,29 @@ const HeroSection: React.FC = () => {
...
+            </div>
+            <div>
+              <TextInput placeholder="Type password here" type="password" />
+            </div>
+          </div>
+
+          <div className="mt-8 flex items-center justify-end space-x-2">
+            <Button size="xs" variant="secondary" onClick={() => setShowLogin(false)}>
```
Comment (golnazsaraji) [inline client/src/components/landing/HeroSection.tsx:40]: I think the button's position is not good, perhaps on the top-right is better!

**Review by golnazsaraji — APPROVED** [review]

**Review by AlbertoPedalino — DISMISSED** [review]
hard to find button due to color

File: client/src/components/landing/HeroSection.tsx (hunk 1 above)
Comment (GiuseppeArbore) [reply client/src/components/landing/HeroSection.tsx:32]: we'll do that in the front-end logic UI

File: client/src/components/landing/HeroSection.tsx (hunk 2 above)
Comment (GiuseppeArbore) [reply client/src/components/landing/HeroSection.tsx:40]: Thank you for the feedback! I tried to apply your suggestion, but I noticed that it affects the symmetry of the code, which slightly impacts its aesthetics.

**Comment by AlbertoPedalino** [comment]
> > hard to find button due to color
> 
> Thank you for your review! The color choice was intentional to make the button less prominent initially, highlighting it only on hover, as it's positioned below the main button in the center of the page. However, if this approach affects user-friendliness I can explore alternative colors to improve visibility. Let me know!

a shade of that blue is fine

**Pull request merged**